module/Company/src/Company/Service/Company.php

```diff
@@ -136,24 +148,15 @@ public function insertCompanyByData($data,$files)
         $companyForm = $this->getCompanyForm();
         $companyForm->setData($data);
         if ($companyForm->isValid()) {
-            //echo "hoi";
-            //var_dump($files);
             $company = $this->insertCompany($data['languages']);
             $company->exchangeArray($data);
-            //echo "hoi";
-            //var_dump($files);
             foreach ($company->getTranslations() as $translation) {
                 $file = $files[$translation->getLanguage() . '_logo'];
-                //var_dump($file);
                 try {
                     $newPath = $this->getFileStorageService()->storeUploadedFile($file);
-                    echo $newPath;
                     $translation->setLogo($newPath);
                 } catch (\Exception $exception) {
```

Comment:
Error message is not needed, because when the upload fails, the logo will not be set, and the next page will display an error, or the user will still see the old image there.